pages/[id].tsx

```diff
@@ -0,0 +1,170 @@
+                    width="38"
+                    height="38"
+                    viewBox="0 0 38 38"
+                    fill="none"
+                    className={styles["oe nz oa"]}
+                  >
+                    <rect
+                      x="26.25"
+                      y="9.25"
+                      width="0.5"
+                      height="6.5"
+                      rx="0.25"
+                      strokeWidth="0.5"
+                    ></rect>
+                    <rect
+                      x="29.75"
+                      y="12.25"
+                      width="0.5"
+                      height="6.5"
+                      rx="0.25"
+                      transform="rotate(90 29.75 12.25)"
+                      strokeWidth="0.5"
+                    ></rect>
+                    <path
+                      d="M19.5 12.5h-7a1 1 0 0 0-1 1v11a1 1 0 0 0 1 1h13a1 1 0 0 0 1-1v-5"
+                      strokeLinecap="round"
+                    ></path>
+                    <path d="M11.5 14.5L19 20l4-3" strokeLinecap="round"></path>
+                  </svg>
+                </button>
+              </div>
+            </div>
+            <div className={styles["related-articles"]}>
+              <h2>Related</h2>
+              <div className={styles["related-items"]}>
+                <div className={styles["related-item"]}>
+                  <img
+                    src="https://miro.medium.com/focal/58/58/50/50/0*SYbbtHZcvbFB0fHR.jpg"
+                    alt="related article"
+                  />
+                  <p>Currying for Javascript Developers with Examples</p>
+                </div>
+                <div className={styles["related-item"]}>
+                  <img
+                    src="https://miro.medium.com/focal/58/58/50/50/0*SYbbtHZcvbFB0fHR.jpg"
+                    alt="related article"
+                  />
+                  <p>Currying for Javascript Developers with Examples</p>
+                </div>
+                <div className={styles["related-item"]}>
+                  <img
+                    src="https://miro.medium.com/focal/58/58/50/50/0*SYbbtHZcvbFB0fHR.jpg"
+                    alt="related article"
+                  />
+                  <p>Currying for Javascript Developers with Examples</p>
+                </div>
+                <div className={styles["related-item"]}>
+                  <img
+                    src="https://miro.medium.com/focal/58/58/50/50/0*SYbbtHZcvbFB0fHR.jpg"
+                    alt="related article"
+                  />
+                  <p>Currying for Javascript Developers with Examples</p>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </Aside>
+    </PageContainer>
+  );
+};
+
+export async function getServerSideProps(context: any) {
+  const { id } = context.params;
+  console.log("context", context);
+  const res = await fetch(`http://localhost:3000/api/articles/${id}`);
```

Comment:
You can replace the base URL with an `env.` variable. That way, you can always easily change it if you have different environments.

Reply:
Resolve Commit: https://github.com/gelo-codeandtheory/dot-fe/pull/3/commits/7ff0ba44eae8af7b3c88b1744724b4feb5193f10

@lycoangelo